packages/utils/src/lib/resource-action-utils.ts

```diff
@@ -50,6 +49,14 @@ export interface ResourceType {
   components: ResourceIdComponent[];
 }
 
+export interface ResourceActionPair {
+  resourceId: string;
+  actionId: string;
+}
+
+export interface GetSpeicificAclIdsResponse {
```

Comment:
Fix typo. And move this to the `access-control-api.ts`. This is specifically only for the API. All API request and response interfaces are supposed to be there.